c2corg_ui/templates/helpers/view.html

```diff
@@ -108,26 +109,33 @@ <h4 translate>No maps associated</h4>
   % if associatedDocuments == 'routes':
   
     % if 'routes' in document['associations'] and  len(document['associations']['routes']) > 0 :
-    <article>
-      <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#associated-${associatedDocuments}">
-        <span class="glyphicon glyphicon-map-marker"></span>
-        <span translate>Associated ${associatedDocuments}</span><span class="glyphicon glyphicon-menu-down"></span>
-      </h3>
-      <div class="associated-documents collapse in" id="associated-${associatedDocuments}">
-        % for route in document['associations']['routes']:
-          ${route_card(route, 'app-delete', document['document_id'])}
-        % endfor
-        <article class="list-item" ng-repeat="doc in added | filter:{'documentType': '${associatedDocuments}'}">
-          <app-association-card parent-id="${document['document_id']}" doc="doc" added-documents="added"></app-association-card>
-        </article>
-      </div>
-    </article>
+      <article>
+        <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#associated-${associatedDocuments}">
+          <span class="glyphicon glyphicon-map-marker"></span>
+          <span translate>Associated ${associatedDocuments}</span><span class="glyphicon glyphicon-menu-down"></span>
+        </h3>
+        <div class="associated-documents collapse in" id="associated-${associatedDocuments}">
+          % for route in document['associations']['routes']:
+          <div class="list-item">
+            ${route_card(route)}
+            <app-delete-association parent-id="${document['document_id']}" child-id="${route['document_id']}"></app-delete-association>
+          </div>
+          % endfor
+          <article class="list-item" ng-repeat="doc in added | filter:{'documentType': '${associatedDocuments}'}">
+            <app-association-card parent-id="${document['document_id']}" doc="doc" added-documents="added"></app-association-card>
+          </article>
+        </div>
+        % if len(document['associations']['routes']) == 10:
```

Comment:
I think we want to show all the associated routes (on the contrary to associated outings).